BUILD.bazel

```diff
@@ -23,106 +23,61 @@ gazelle(
     srcs = [
-        "//:install-gapir",
-        "//:install-gapis",
-        "//:install-properties",
-        "//:install-strings",
-    ] + select({
-        # TODO: temporary until android rules work on windows
-        "//tools/build:windows": [],
-        "//conditions:default": ["//:install-gapid.apk"],
-    }),
-    tags = ["manual"],
-    visibility = ["//visibility:public"],
-)
-
-filegroup(
-    name = "install-gapid.apk",
-    srcs = select({
-        "//tools/build:config-armeabi": [":install-armeabi_gapid.apk"],
-        "//tools/build:config-x86": [":install-x86_gapid.apk"],
-        "//tools/build:config-aarch64": [":install-aarch64_gapid.apk"],
-    }),
-    tags = ["manual"],
-    visibility = ["//visibility:public"],
+        "//tools/build:build.properties",
+        "//cmd/gapid",
+        "//cmd/gapir/cc:gapir",
+        "//cmd/gapis",
+        "//cmd/gapit",
+        "//gapidapk/android/apk:armeabi-v7a.apk",
+        "//gapidapk/android/apk:arm64-v8a.apk",
+        "//gapidapk/android/apk:x86.apk",
+    ],
+    rename = {
+        "armeabi-v7a.apk": "gapid-armeabi.apk",
+        "arm64-v8a.apk": "gapid-aarch64.apk",
+        "x86.apk": "gapid-x86.apk",
+    },
+    to = "pkg",
 )
 
-filegroup(
-    name = "install-strings",
+copy_to(
+    name = "install-pkg-lib",
     srcs = [
-        ":install-strings-en-us",
+        "//gapic:gapic_deploy.jar",
+        "//gapii/vulkan/vk_graphics_spy/cc:GraphicsSpyLayer.json",
+        "//gapii/cc:gapii",
+        "//core/vulkan/vk_virtual_swapchain/cc:cc",
+        "//core/vulkan/vk_virtual_swapchain/cc:VirtualSwapchainLayer.json",
     ],
-    tags = ["manual"],
-    visibility = ["//visibility:public"],
-)
-
-copy_platform_binaries(
-    name = "install-gapic",
-    src = "//gapic",
-    tags = ["manual"],
-    visibility = ["//visibility:public"],
-)
-
-copy_platform_binaries(
-    name = "install-gapir",
-    src = "//cmd/gapir/cc:gapir",
-    tags = ["manual"],
-    visibility = ["//visibility:public"],
-)
-
-copy_platform_binaries(
-    name = "install-gapis",
-    src = "//cmd/gapis",
-    tags = ["manual"],
-    visibility = ["//visibility:public"],
-)
-
-copy_platform_binaries(
-    name = "install-gapit",
-    src = "//cmd/gapit",
-    tags = ["manual"],
-    visibility = ["//visibility:public"],
-)
-
-copy(
-    name = "install-armeabi_gapid.apk",
-    src = "//gapidapk/android/apk:armeabi.apk",
-    dst = "android/armeabi-v7a/gapid.apk",
-    tags = ["manual"],
-    visibility = ["//visibility:private"],
-)
-
-copy(
-    name = "install-x86_gapid.apk",
-    src = "//gapidapk/android/apk:x86.apk",
-    dst = "android/x86/gapid.apk",
-    tags = ["manual"],
-    visibility = ["//visibility:private"],
-)
-
-copy(
-    name = "install-strings-en-us",
-    src = "//gapis/messages:stb",
-    dst = "strings/en-us.stb",
-    tags = ["manual"],
-    visibility = ["//visibility:private"],
+    extensions = ["jar", "json", "so"],
+    # Unfortunatly, no dict concatination support.
```

Comment:
It is possible to add dictionaries by abusing the dict function...
dict({a:"A"}, **{b:"B"})